util/crashReport.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+# TODO
+# remove 'logging' echo statements
+
+# Variables for script usage
+FILE_HS_ERR_LOG=/opt/instana/agent/hs_err.log
+DIR_AGENT_LOG=/opt/instana/agent/data/log
+AGENT_LOG_COLLECT_LINES=1000
+
+# Variables for use in the JSON payload
+JSON_HOSTNAME="$(hostname)"
```

Comment:
While the Agent itself has more advanced logic to determine the hostname, it would be hard to replicate that here and at least initially the assumption is this would be good enough.